content/post/benchmarking-gnns/index.md

```diff
@@ -0,0 +1,249 @@
+
+---
+
+### Why benchmark?
+In any core research or application area in [deep learning](https://www.nature.com/articles/nature14539), a benchmark helps to identify and quantify what types of [architectures](https://arxiv.org/abs/1409.4842), [principles](https://arxiv.org/abs/1512.03385), or [mechanisms](https://arxiv.org/abs/1502.03167) are universal and generalizable to real-world tasks and large datasets. Particularly, the [recent](https://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf) [revolution](https://cacm.acm.org/magazines/2017/6/217744-technical-perspective-what-led-computer-vision-to-deep-learning/fulltext) in this AI field is often credited, _to a possibly large extent_, to be triggered by the large-scale benchmark image dataset, [ImageNet](http://www.image-net.org). [Obviously, other driving factors include increase in the volume of research, more datasets, compute, wide-adoptance, etc.]
+
+{{< figure src="imagenet_leaderboard.png" title="Fig 1: ImageNet Classification Leaderboard from [paperswithcode.com](https://paperswithcode.com)" lightbox="true" width="100%">}}
+
+Benchmarking has been proved to be beneficial for **driving progress**, identifying **essential ideas**, and solving domain-related problems in [many](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8) sub-fields of science. This project was conceived with this fundamental motivation.
+
+---
+
+### Need of a benchmarking framework for GNNs
+
+#### a. Datasets:
+Many of the widely cited papers in the GNN literature contain experiments that are evaluated on **small graph datasets** which have only hundreds (or, few thousand graphs). 
+
+{{< figure src="tu_datasets.png" title="Fig 2: Statistics of the widely used TU datasets. Source [Errica et al., 2020](https://openreview.net/forum?id=HygDF6NFPB)" lightbox="true" width="100%">}}
+
+>_Take for example_, the ENZYMES dataset, which is almost seen in every work on a GNN for classification task. If one uses a random $10$-fold cross validation (in most papers), the test set would have $60$ graphs (i.e. $10$% of $600$ total graphs). That would mean a correct classification (or, alternatively a misclassification) would change $1.67$% of test accuracy score. **A couple of samples could determine a $3.33$% difference in performance measure**, which is usually a significant gain score stated when one validates a new idea in literature. You see there, the number of samples is unreliable to concretely acknowledge the advances. ^[By this, we do not mean the ideas are not useful and/or the work put by the authors is not meaningful. Every effort equally contributes to the advance of this field.]^ 
```

Comment:
Seems some reformatting is needed for the first few words. It is not rendering as italicized text on my screen. Neither is the bolded text rendering as bold. Maybe you will have to use html tags instead of markdown (not sure what is the reason).

Reply:
Is the '^[ ... ]^' supposed to render some different type of braces, too? It is also not rendering.

Reply:
My target to have ^[]^ is to place the sentence in the footnote.

The footnote and italics worked fine on my local server, but it doesnt work online. I will try resolving, maybe using html.